core/server/proxy/src/test/java/alluxio/StreamCacheTest.java

```diff
@@ -34,11 +36,78 @@ public void operations() throws Exception {
     Assert.assertSame(os, streamCache.invalidate(osId));
     Assert.assertNull(streamCache.invalidate(isId));
     Assert.assertNull(streamCache.invalidate(osId));
+    Assert.assertNull(streamCache.getInStream(isId));
+    Assert.assertNull(streamCache.getOutStream(osId));
     Mockito.verify(is).close();
     Mockito.verify(os).close();
   }
 
   @Test
+  public void concurrentOperations() throws Exception {
+    final StreamCache streamCache = new StreamCache(Constants.HOUR_MS);
+    final FileInStream is = Mockito.mock(FileInStream.class);
+    final FileOutStream os = Mockito.mock(FileOutStream.class);
+
+    // Concurrent put.
```

Comment:
can we perform 100 concurrent puts to both input streams and output streams instead of just one?